azure-iot-device/azure/iot/device/provisioning/abstract_provisioning_device_client.py

```diff
@@ -103,6 +104,15 @@ def cancel(self):
         """
         pass
 
+    def set_provisioning_payload(self, custom_payload):
```

Comment:
>set_provisioning_payload [](start = 8, length = 24)

Use a @property decorator instead.

Or better yet, just let them set the attribute directly

Reply:
its now a property

---
In reply to: [335731680](https://github.com/Azure/azure-iot-sdk-python/pull/351#discussion_r335731680) [](ancestors = 335731680)